sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Mode.scala

```diff
@@ -201,7 +212,8 @@ case class PandasMode(
     child: Expression,
     ignoreNA: Boolean = true,
     mutableAggBufferOffset: Int = 0,
-    inputAggBufferOffset: Int = 0) extends TypedAggregateWithHashMapAsBuffer
+    inputAggBufferOffset: Int = 0)
+  extends TypedAggregateWithHashMapAsBuffer
```

Comment:
looks like an unnecessary change, but I suppose it's a prototyping leftover so I won't comment these more for now